en/news/_posts/2019-10-22-ruby-2-7-0-preview2-released.md

```diff
@@ -0,0 +1,244 @@
+
+      ["a", "b", "c", "b"].tally
+      #=> {"a"=>1, "b"=>2, "c"=>1}
+
+* Calling a private method on `self` is now allowed.  [Feature #11297] [Feature #16123]
+
+      def foo
+      end
+      private :foo
+      self.foo
+
+* `Enumerator::Lazy#eager` is added.  It generates a non-lazy enumerator
+  from a lazy enumerator.  [Feature #15901]
+
+      a = %w(foo bar baz)
+      e = a.lazy.map {|x| x.upcase }.map {|x| x + "!" }.eager
+      p e.class               #=> Enumerator
+      p e.map {|x| x + "?" }  #=> ["FOO!?", "BAR!?", "BAZ!?"]
+
+## Performance improvements
+
+* JIT [Experimental]
+
+  * JIT-ed code is recompiled to less-optimized code when an optimization assumption is invalidated.
+
+  * Method inlining is performed when a method is considered as pure. This optimization is still experimental and many methods are NOT considered as pure yet.
+
+  * Default value of `--jit-min-calls` is changed from 5 to 10,000
+
+  * Default value of `--jit-max-cache` is changed from 1,000 to 100
+
+* `Symbol#to_s`, `Module#name`, `true.to_s`, `false.to_s` and `nil.to_s` now always return a frozen String. The returned String is always the same for a given object. [Experimental]  [[Feature #16150]](https://bugs.ruby-lang.org/issues/16150)
+
+* The performance of CGI.escapeHTML is improved.
+  * https://github.com/ruby/ruby/pull/2226
+
+## Other notable changes since 2.6
+
+* Some standard libraries are updated.
+  * Bundler 2.1.0.pre.1
+  * RubyGems 3.1.0.pre.1
+  * CSV 3.1.2 ([NEWS](https://github.com/ruby/csv/blob/master/NEWS.md))
+  * Racc 1.4.15
+  * REXML 3.2.3 ([NEWS](https://github.com/ruby/rexml/blob/master/NEWS.md))
+  * RSS 0.2.8 ([NEWS](https://github.com/ruby/rss/blob/master/NEWS.md))
+  * StringScanner 1.0.3
+  * Some of other libraries that have no original version are also updated.
+
+* `Proc.new` and `proc` with no block in a method called with a block is warned now.
+
+* `lambda` with no block in a method called with a block errs.
+
+* Update Unicode version and Emoji version from 11.0.0 to 12.0.0.  [[Feature #15321]](https://bugs.ruby-lang.org/issues/15321)
+
+* Update Unicode version to 12.1.0, adding support for U+32FF SQUARE ERA NAME REIWA.  [[Feature #15195]](https://bugs.ruby-lang.org/issues/15195)
+
+* `Date.jisx0301`, `Date#jisx0301`, and `Date.parse` provisionally support the new Japanese era as an informal extension, until the new JIS X 0301 is issued.  [[Feature #15742]](https://bugs.ruby-lang.org/issues/15742)
+
+* Require compilers to support C99 [[Misc #15347]](https://bugs.ruby-lang.org/issues/15347)
+  * Details of our dialect: <https://bugs.ruby-lang.org/projects/ruby-trunk/wiki/C99>
+
+See [NEWS](https://github.com/ruby/ruby/blob/v2_7_0_preview1/NEWS) or [commit logs](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview1) for more details.
+
+With those changes, [1727 files changed, 76022 insertions(+), 60286 deletions(-)](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview1) since Ruby 2.6.0!
```

Comment:
```suggestion
With those changes, [3670 files changed, 201242 insertions(+), 88066 deletions(-)](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview2) since Ruby 2.6.0!
```